pkg/k8s/apis/cilium.io/v2/client/register.go

```diff
@@ -279,6 +279,7 @@ func createNodeCRD(clientset apiextensionsclient.Interface) error {
 					Description: "CiliumNode represents the k8s node from the view of Cilium.",
 					Properties: map[string]apiextensionsv1beta1.JSONSchemaProps{
 						"spec": {
+							Type: "object",
```

Comment:
It's not as easy has this. Check lines 160-165 of this file.

Reply:
I think lines 160-165 apply to `spec.Validation.OpenAPIV3Schema.Type`. This PR is to fix `spec.Validation.OpenAPIV3Schema.Properties.spec.Type`.

Reply:
oh, you are right. Cool!